<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>twoplayers</class>
 <widget class="QMainWindow" name="twoplayers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background:brgb(0, 0, 0)

#gameSecreen{
 background-image: url(&quot;:/menu_back.jpg&quot;);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-190</x>
      <y>-50</y>
      <width>1311</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame {
background-image:url(:/menu_back.jpg);}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>688</x>
       <y>183</y>
       <width>541</width>
       <height>401</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="button1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">
#button1
{ 
background-image:  url(&quot;:/rose.jpeg&quot;);
color : white;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon1" stdset="0">
         <iconset>
          <normaloff>:/R.jfif</normaloff>:/R.jfif</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="button3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#button3

{ 
background-image:  url(&quot;:/rose.jpeg&quot;);
color : white;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="button2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#button2

{ 
background-image:  url(&quot;:/rose.jpeg&quot;);
color : white;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="button8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#button8

{ 
background-image:  url(&quot;:/rose.jpeg&quot;);
color : white;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="button9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#button9

{ 
background-image:  url(&quot;:/rose.jpeg&quot;);
color : white;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="button7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#button7
{ 
background-image:  url(&quot;:/rose.jpeg&quot;);
color : white;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="button6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#button6
{ 
background-image:  url(&quot;:/rose.jpeg&quot;);
color : white;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="button5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#button5
{ 
background-image:  url(&quot;:/rose.jpeg&quot;);
color : white;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="button4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#button4
{ 
background-image:  url(&quot;:/rose.jpeg&quot;);
color : white;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>590</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>590</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton {
background-color :qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(221, 0, 255, 69), stop:0.363636 rgba(229, 169, 255, 130), stop:0.375 rgba(186, 0, 255, 69), stop:0.423533 rgba(233, 119, 255, 145), stop:0.45 rgba(255, 121, 250, 208), stop:0.477581 rgba(255, 71, 241, 130), stop:0.518717 rgba(250, 71, 255, 130), stop:0.55 rgba(255, 87, 214, 255), stop:0.57754 rgba(239, 0, 255, 130), stop:0.625 rgba(255, 86, 237, 69), stop:1 rgba(255, 255, 0, 69))}</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>751</width>
       <height>129</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Viner Hand ITC</family>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label{

font-color : rgb(176, 0, 176)}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#914479;&quot;&gt;TIC TAC TOE &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
